project.clj

```diff
@@ -13,8 +13,9 @@
 
   :dependencies
   [[org.clojure/clojure "1.12.0" :scope "provided"]
-   [org.bouncycastle/bcpg-jdk18on "1.79"]
-   [org.bouncycastle/bcprov-jdk18on "1.79"]
+   [org.bouncycastle/bcpg-jdk18on "1.80" :exclusions [org.bouncycastle/bcprov-jdk18on]]
+   [org.bouncycastle/bcutil-jdk18on "1.80" :exclusions [org.bouncycastle/bcprov-jdk18on]]
+   [org.bouncycastle/bcprov-jdk18on "1.80"]
```

Comment:
Was going to ask why this was necessary, but now I see that both packages specify a range for `bcprov-jdk18on` of `[1.80,1.81)` which is really annoying.